File: app/src/main/java/ru/beerproject/vezdekodravens/MainActivity.java
package ru.beerproject.vezdekodravens;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.ScrollView;

import me.everything.android.ui.overscroll.OverScrollDecoratorHelper;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_target);
    }
}

File: app/src/main/java/ru/beerproject/vezdekodravens/MainActivity.kt
package ru.beerproject.vezdekodravens

import android.content.Intent
import android.os.Bundle
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity
import com.google.android.material.transition.MaterialSharedAxis
import kotlinx.android.synthetic.main.activity_main.*
import kotlinx.android.synthetic.main.activity_type_select.*

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        x_base_butt.setOnClickListener{
            val intent = Intent(this, TypeSelectActivity::class.java)
            startActivity(intent)
        }
    }
}